controllers/news.js

```diff
@@ -0,0 +1,31 @@
+const createHttpError = require('http-errors')
+const { endpointResponse } = require('../helpers/success')
+const { getNewById } = require('../services/news')
+
+const getNews = async (req, res, next) => {
+  try {
+    const { id } = req.Params
+    if (id) {
+      const result = await getNewById(id)
+      if (!result) {
+        next(createHttpError(404, 'New not found'))
+      }
+      endpointResponse({
+        res,
+        code: 200,
+        status: true,
+        message: result,
+      })
+    }
+  } catch (e) {
```

Comment:
```suggestion
  } catch (error) {
```
mismo aca, al error lo vamos a llamar error

Reply:
+1

Reply:
Modificado